include/openthread/ip6.h

```diff
@@ -97,6 +97,18 @@ struct otIp6Prefix
  */
 typedef struct otIp6Prefix otIp6Prefix;
 
+/**
+ * IPv6 Address origins
+ *
+ */
+enum
+{
+    OT_ADDRESS_ORIGIN_SYSTEM = 0, ///< Address assigned by the system
```

Comment:
nit: can we be more specific than "system"?
```suggestion
    OT_ADDRESS_ORIGIN_SYSTEM = 0, ///< Address assigned by Thread control protocols
```

Reply:
I'd prefer:
```suggestion
    OT_ADDRESS_ORIGIN_SYSTEM = 0, ///< Thread assigned address (ALOC, RLOC, MLEID, etc)

```

Reply:
> nit: can we be more specific than "the system"?

I think `OT_ADDRESS_ORIGIN_THREAD` might be a more suitable name. Thoughts?

Reply:
@markus-becker-tridonic-com , "Thread assigned address" sounds good to me.

Reply:
done